patches/chrome-chrome_repack_locales.gni.patch

```diff
@@ -18,3 +18,14 @@ index 7aa8989a0f47167440824f0531ee9956d2f24098..d7a9da068994a9450eca2655dd5c038b
      if (defined(invoker.deps)) {
        deps += invoker.deps
      }
+@@ -106,8 +108,8 @@ template("chrome_repack_locales") {
+       ]
+     } else {
+       source_patterns += [
+-        "${root_gen_dir}/chrome/chromium_strings_",
```

Comment:
please use `sources_patterns -= [ .. ] source_patterns += [ ... ]` to keep this one line and not have deletions

Reply:
actually this should be submitted as an upstream patch

Reply:
I'll submit a CL tomorrow for these cases.